src/components/Map.tsx

```diff
@@ -229,13 +256,17 @@ export class MapCanvas implements GestureTarget {
     }
 
     onClick(coord: ClientCoordinates) {
+        coord = this.clientToCanvas(coord);
         let data = null
+        
         switch (this.tool) {
             case MapTools.Stamp:
                 data = 1
+                this.map.setTile(this.canvasToMap(coord.clientX - this.offsetX), this.canvasToMap(coord.clientY - this.offsetY), 1);
```

Comment:
This shouldn't be here, this might be the artifact of a bad merge. The way we mutate maps now is to use the "triggerOperation" method that is already called below. This way we can undo/redo map edits.